_posts/2018-10-30-conducting-good-retrospectives.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: Conducting Good Retrospectives
+date: 2018-10-30 09:00:00 -04:00
+categories: post
+desc: Introduction to holding wildly productive and efficient team retrospectives.
+permalink: /:categories/:year/:month/:day/:title/
+image: https://res.cloudinary.com/mhanberg/image/upload/t_scale/v1541683676/jametlene-reskp-651411-unsplash.jpg
+image_desc: Photo by Jametlene Reskp on Unsplash
+---
+
+Your team has probably experienced moments of success or failure, moments of working like a well-oiled-machine or not being able to stand the sound of each other's voices. What you've probably also done is pass these moments by without taking time to ponder "Why did these things happen"?
+
+This is when your team holds a __retrospective__.
+
+### What is a retrospective?
+
+A retrospective (or "retro") is a recurring meeting used by teams to look back at the last week or two in order to figure out what went well and what went poorly. The realm of topics is wide ranging: from missing deadlines to mishandled social interactions. These meetings aren't just to make you feel good or bad about yourselves, they're designed to curate items on which your team will take action after the meeting.
+
+Setting aside dedicated time for these matters assures that they will get some attention and helps keep these conversations from spilling into normal work time (this isn't to say that you shouldn't discuss something when it happens).
+
+Along with **learning from your successes and failures**, one of the goals of a retrospective to **build trust amongst your team**. Teams perform better when they operate in an environment of [psychological safety](https://hbr.org/2017/08/high-performing-teams-need-psychological-safety-heres-how-to-create-it).
+
+## How to conduct a retrospective
+
+Anyone can host a retrospective, but hosting a _good_ retrospective takes some finesse. There are four key aspects that I believe are paramount to a good retrospective: Facilitation, Conversational Turn Taking, Format/Activity, and Action Items.
+
+#### Facilitation
+
+Someone should volunteer to facilitate the meeting. This person will lead the chosen activity for the meeting. Depending on the activity, this can include: keeping a conversation timer, taking notes, or tracking whose turn it is to speak. 
```

Comment:
Is it worth noting that these activities can be done by different people - or does that just muddy the waters?